doc/vxlan/EVPN/EVPN_VXLAN_HLD.md

```diff
@@ -0,0 +1,1235 @@
+    "remote_vtep" : {{ipv4_address}}
+    "type"   : "static" / "dynamic"  ; type of the entry.
+    "vni" : {{vni_id}}
+```
+
+In the case of MAC move from remote to local, remote MAC entry will be removed by Zebra from Linux FDB. Fdbsyncd will remove the remote mac from APP_VXLAN_FDB_TABLE when MAC moves.
+
+#### 4.3.7.1 Local MAC handling
+
+There is no change to fdborch wrt local MAC handling. 
+
+Currently Local MACs learnt by the fdborch are written to the STATE_DB. 
+
+Fdbsyncd listens to the above table and notifies Linux FDB which notifies to Zebra, Zebra to BGP and BGP originates the MAC route. 
+
+The above is applicable for locally added static MACs also. 
+
+![Local MAC Handling](images/localmac.PNG "Figure : Local MAC handling")
+__Figure 9: Local MAC handling__
+
+#### 4.3.7.2 Remote MAC handling
+
+Remote MACs learnt by BGP will be installed in Linux FDB via Zebra; Fdbsyncd will be listening to Linux FDB, MAC learned over Vxlan interface will be populated in APP_VXLAN_FDB_TABLE.
+
+fdborch does the following on an entry add in APP_VXLAN_FDB_TABLE. 
+
+- Store the Remote MAC in an internal cache. The internal cache used for local MACs will be used for storing remote MACs as well. 
+- sai_fdb_entry_add called with the following attributes
+  - SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID with the bridge port id corresponding to the tunnel.
+  - SAI_FDB_ENTRY_ATTR_ENDPOINT_IP with the tunnel DIP.
+  - SAI_FDB_ENTRY_ATTR_TYPE with type SAI_FDB_ENTRY_TYPE_STATIC_MACMOVE (corresponding to type dynamic in VXLAN_FDB_TABLE)  or SAI_FDB_ENTRY_TYPE_STATIC (corresponding to type static in VXLAN_FDB_TABLE)  
+
+![Remote MAC Handling](images/remotemac.PNG "Figure : Remote MAC handling")
+__Figure 10: Remote MAC handling__
+
+
+#### 4.3.7.3 MAC Move handling
+
+##### Local to Local
+
+There is no special handling by SwSS. 
+
+Local to Local MAC moves will be identified by BGP and no MAC route will be re-originated. 
+
+##### Remote to Local
+
+For MAC move cases from remote to local, fdbsyncd removes the 
+remote entry from VXLAN_FDB_TABLE. The fdborch should not delete the 
+MAC from the internal cache and the hardware. 
+
+##### Local to Remote
+
+MAC move from local to remote tunnel is handled as follows.
+
+- fdbsyncd adds the new remote MAC entry to the APP_VXLAN_FDB_TABLE.
+- fdborch looks up internal cache to see if an entry exists. If it exists with a local port as the destination then the entry is deleted and the new entry populated.
+- If the local MAC is configured as static then the received remote MAC will not be programmed and instead will be marked as pending. This will be programmed when the local static MAC is removed. 
+
+##### Remote to Remote
+
+MAC Move from remote tunnel-1 to remote tunnel-2 is handled as follows. 
+
+- On identifying a MAC Move, BGP deletes the old entry from APP_VXLAN_FDB_TABLE and adds the new entry pointing to the tunnel-2.
+- fdborch processing is same as for a regular remote MAC add/delete.  
```

Comment:
is there any section for the remote MAC delete flow.  I only see the new Remote MAC flow